libsolidity/interface/SMTSolverCommand.cpp

```diff
@@ -37,7 +37,16 @@ using solidity::util::errinfo_comment;
 namespace solidity::frontend
 {
 
-SMTSolverCommand::SMTSolverCommand(std::string _solverCmd) : m_solverCmd(_solverCmd) {}
+void SMTSolverCommand::setEldarica(std::optional<unsigned int> timeoutInMilliseconds) {
```

Comment:
```suggestion
void SMTSolverCommand::setEldarica(std::optional<unsigned int> timeoutInMilliseconds)
{
```
strictly speaking per our coding style

Reply:
Fixed.